packages/vscode/languages/astro-language-configuration.json

```diff
@@ -3,7 +3,6 @@
     "blockComment": ["<!--", "-->"]
   },
   "brackets": [
-    ["---", "---"],
```

Comment:
Removed the frontmatter bracket because it causes an indent bump into the template (since the entry and the exit of the bracket are the same characters). If we want to provide an indent into the frontmatter, we should use entry rules